routers/repo/issue.go

```diff
@@ -1705,3 +1654,58 @@ func GetCommentAttachments(ctx *context.Context) {
 	}
 	ctx.JSON(200, attachments)
 }
+
+func updateAttachments(item interface{}, files []string) error {
+	var attachments []*models.Attachment
+	switch content := item.(type) {
+	case *models.Issue:
+		attachments = content.Attachments
```

Comment:
Isn't it better to use an explicit interface? I _think_ the syntax looks something like this:
```golang
func updateAttachments(item interface{Attachments []*Attachment,  UpdateAttachments func([]string) error, files []string) error {
```

Reply:
> 
> 
> Isn't it better to use an explicit interface? I _think_ the syntax looks something like this:
> 
It won't work, sorry. Disregard my comment, please.